### PR TITLE
[14.0][FIX] stock_location_orderpoint_source_relocate

### PR DESCRIPTION
**Pull request by jbaudoux**
Check orderpoints also for non relocated moves
Fix quantity to replenish in case of partial relocation (move partially available)

Depends on:
* https://github.com/OCA/wms/pull/755

cc @mt-software-de @rousseldenis

### CODE REVIEW

**Comment by mt-software-de** [comment]
Could you push your latest commits?

**Comment by mt-software-de** [comment]
> Could you push your latest commits?
Ok the merge of  https://github.com/OCA/wms/pull/755 failed. thats why the tests are failing

File: stock_location_orderpoint_source_relocate/tests/test_location_orderpoint.py [hunk 1]
```diff
@@ -11,48 +11,110 @@
...
+        cls.orderpoint = cls.orderpoint.save()
 
-        orderpoint = Form(self.env["stock.location.orderpoint"])
-        orderpoint.location_id = internal_location
-        orderpoint.route_id = route
-        orderpoint = orderpoint.save()
+        cls._create_incoming_move(10, cls.replenishment_location)
 
+    def test_auto_replenishment_without_relocation(self):
+        job_func = self.env["stock.location.orderpoint"].run_auto_replenishment
```
Comment (mt-software-de) [inline stock_location_orderpoint_source_relocate/tests/test_location_orderpoint.py]:
```suggestion
        cls._create_incoming_move(10, cls.replenishment_location)
        cls.job_func = cls.env["stock.location.orderpoint"].run_auto_replenishment

    def test_auto_replenishment_without_relocation(self):
        job_func = self.job_func
```
Could you move this also to the setUpClass?

**Review by mt-software-de — APPROVED** [review]
Only minor remarks. LGTM

File: stock_location_orderpoint_source_relocate/tests/test_location_orderpoint.py (hunk 1 above)
Comment (jbaudoux) [reply stock_location_orderpoint_source_relocate/tests/test_location_orderpoint.py]: done

**Review by mt-software-de — APPROVED** [review]

**Pull request merged**